app/views/articles/show.html.erb

```diff
@@ -1,11 +1,42 @@
 <main class="page-content">
   <div class="wrapper">
+    <%= render partial: 'shared/flash' %>
     <h1><%= @article.title %></h1>
     <%= @article.body.html_safe %>
     <% if @article.content_img.attached? %>
       <img src='<%= url_for(@article.content_img) %>' class='article-img'/>
     <% end %>
-    <div>
+    
+    <div class="form-element form-comment">
+      <% if current_user %>
+        <%= form_for @comment, url: article_comments_path(@article), html: {class: 'new-form'} do |form| %>
+          <p class="form-p">
+            <%= form.label :body, 'Enter comment:' %>
+            <%= form.cktext_area :body, class: 'textarea-field', ckeditor: { uiColor: '#ffe6bd', toolbar: 'mini', height: 100, width: 400 } %>
+            <!-- 
+    @source https://cdn.ckeditor.com/#ckeditor5
+    -->
+            <script type="text/javascript">
+              createReplyToolBar();
+            </script> 
+          </p>
+
+          <p class="form-p">
+            <%= form.submit 'Reply', class: 'comment-reply' %>
+          </p>
+        <% end %>
+      <% end %>
+      <hr>
+      
+      <% unless @article.comments.count == 0 %>
```

Comment:
minor suggestion: could use Ruby's `.zero?` method for a fun reading experience.

Reply:
Actually, the zero? method is undefined for a model object but the empty? works

Reply:
`zero?` is defined on `Integer` or `Number` or one of those Ruby classes, so you'd call it on `count` here. `empty?` might work better; depends on the SQL query executed.